sms_sponsorship/webapp/src/i18n.js

```diff
@@ -35,7 +36,6 @@ i18n
                     "emailConfirm": "Confirm your email address",
                     "sponsorshipPlus": "Sponsorship plus",
                     "otherChild": "Other child",
-                    "newSponsor": "New sponsor",
```

Comment:
Why is this string removed?

Reply:
This text was related to the "new sponsor" button that I propose to remove
it only reset the content of the form. If you found it mandatory I would suggest to change the button's text as it is a bit confusing imo

Reply:
OK